Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -29,6 +29,7 @@ public class FreezeOnMatchCustomization : ICustomization
         /// <exception cref="ArgumentNullException">
         /// <paramref name="targetType"/> is <see langword="null"/>.
         /// </exception>
+        [Obsolete("Please use FreezeOnMatchCustomization(object, IRequestSpecification) instead.")]
```

Comment:
Wouldn't it be a better user experience if we could add a constructor overload that doesn't demand of the user that he or she supplies an explicit `IRequestSpecification`?